onfido-java/src/main/java/com/onfido/api/Resources.java

```diff
@@ -0,0 +1,113 @@
+package com.onfido.api;
+
+import java.io.IOException;
+import java.util.Map;
+
+import com.onfido.Onfido;
+import com.onfido.exceptions.ApiException;
+import com.onfido.exceptions.OnfidoException;
+
+import okhttp3.MediaType;
+import okhttp3.MultipartBody;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import okio.BufferedSink;
+import okio.Okio;
+import okio.Source;
+
+public class Resources {
```

Comment:
I'd be tempted to rename this to `ResourceManager` for consistency with its subclasses names

Reply:
Done